src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -52,6 +54,16 @@ const ActionSidebar: FC<PropsWithChildren> = ({ children }) => {
             title={{ id: 'actionSidebar.mint.token.permission.error' }}
           />
         )}
+        {selectedAction === Actions.UNLOCK_TOKEN && (
+          <div className="mb-7">
+            <NotificationBanner
```

Comment:
the notification banner is already implemented in 53 line. Please pass there all neeaded props and reuse it.